recipes/chaiscript/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+project(test_package CXX)
+cmake_minimum_required(VERSION 2.8.12)
+
+set(CMAKE_VERBOSE_MAKEFILE ON)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+find_package(chaiscript REQUIRED CONFIG)
+find_package(Threads)
```

Comment:
Why do you need it if you are exporting pthread in package_info?

Reply:
I'm still learning the conan's way...